pyccel/stdlib/ndarrays/ndarrays.c

```diff
@@ -192,7 +202,7 @@ t_ndarray array_slicing(t_ndarray p, ...)
     slice.shape = malloc(sizeof(int32_t) * p.nd);
     slice.strides = malloc(sizeof(int32_t) * p.nd);
     memcpy(slice.strides, p.strides, sizeof(int32_t) * p.nd);
-    slice.is_slice = true;
+    slice.is_view = true;
     va_start(va, p);

```

Comment:
Sorry to ask for this, but I must have missed it in the previous PR. I think it is **really** confusing to call `slice` an array variable, and then call `slice_data` a slice variable. Can you call them `view` (or `arr`) and `slice` please?
```C
t_ndarray view;
t_slice slice;
```
If possible, I would like to use the same names throughout this whole file. We can chat about this.

Reply:
i will try to start only using (arr , view, slice) throughout the whole file.

Reply:
Thanks!